File: java/src/main/java/vsdk/toolkit/io/context/MaterialSelectionContext.java
package vsdk.toolkit.io.context;

import java.util.ArrayList;
import vsdk.toolkit.material.Material;

public class MaterialSelectionContext {
    public Material currentMaterial;
    public String currentMaterialName;
    public ArrayList<Material> materials;

    public MaterialSelectionContext() {
        currentMaterial = null;
        currentMaterialName = null;
        materials = null;
    }
}

File: typescript/base/src/main/vsdk/toolkit/io/context/MaterialSelectionContext.ts
import { Material } from "../../material/Material";

export class MaterialSelectionContext {
  public currentMaterial: Material | null;
  public currentMaterialName: string | null;
  public materials: Material[] | null;

  public constructor() {
    this.currentMaterial = null;
    this.currentMaterialName = null;
    this.materials = null;
  }
}
